### ISSUE TITLE
Extension Error: Expected expectedValue of type boolean, got number: Unabl...

### ISSUE THREAD

**Issue by mourawaldson** [GoogleChrome/lighthouse#1625]
**Lighthouse Version**: 1.4.1
**Chrome Version**: 56.0.2924.87
**URL**: https://app.ft.com/index_page/home
**Error Message**: Expected expectedValue of type boolean, got number: Unable to get document body innerText: an unexpected driver error occurred
**Stack Trace**:
 ```Error: Expected expectedValue of type boolean, got number: Unable to get document body innerText: an unexpected driver error occurred
    at Function._convertToWeight (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:5641:13)
    at expectedNames.forEach.e (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:5735:35)
    at Array.forEach (native)
    at items.map.item (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:5708:21)
    at Array.map (native)
    at Function.compare (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:5692:18)
    at Function.aggregate (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:5770:29)
    at config.aggregations.map.a (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:11565:28)
    at Array.map (native)
    at run.then.runResults (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:11564:46)```

**Comment by brendankenny** [comment]
this will be fixed by #1623

**Comment by ebidel** [comment]
Marking this as fixed. It'll be available in the next release.
